diff --git a/src/OpenFga.Sdk/Model/TupleKey.cs b/src/OpenFga.Sdk/Model/TupleKey.cs
index e229ef1..73b118a 100644
--- a/src/OpenFga.Sdk/Model/TupleKey.cs
+++ b/src/OpenFga.Sdk/Model/TupleKey.cs
@@ -40,6 +40,15 @@ namespace OpenFga.Sdk.Model
             this.AdditionalProperties = new Dictionary<string, object>();
         }

+        /// <summary>
+        /// Initializes a new instance of the <see cref="TupleKey" /> class.
+        /// </summary>
+        [JsonConstructor]
+        public TupleKey()
+        {
+            this.AdditionalProperties = new Dictionary<string, object>();
+        }
+
         /// <summary>
         /// Gets or Sets Object
         /// </summary>