diff --git a/src/OpenFga.Sdk/Model/Nodes.cs b/src/OpenFga.Sdk/Model/Nodes.cs
index a96b2d7..8c42aa8 100644
--- a/src/OpenFga.Sdk/Model/Nodes.cs
+++ b/src/OpenFga.Sdk/Model/Nodes.cs
@@ -36,6 +36,15 @@ namespace OpenFga.Sdk.Model
             this.AdditionalProperties = new Dictionary<string, object>();
         }

+        /// <summary>
+        /// Initializes a new instance of the <see cref="Nodes" /> class.
+        /// </summary>
+        [JsonConstructor]
+        public Nodes()
+        {
+            this.AdditionalProperties = new Dictionary<string, object>();
+        }
+
         /// <summary>
         /// Gets or Sets _Nodes
         /// </summary>