--- clients/fga-dotnet-sdk/src/OpenFga.Sdk/Api/OpenFgaApi.cs	2022-11-18 13:11:05.000000000 -0500
+++ OpenFgaApi.cs	2022-11-18 13:12:35.000000000 -0500
@@ -76,9 +76,10 @@
     /// Create a store Create a unique OpenFGA store which will be used to store authorization models and relationship tuples.
     /// </summary>
     /// <exception cref="OpenFga.Sdk.Exceptions.ApiException">Thrown when fails to make API call</exception>
+    /// <param name="body"></param>
     /// <param name="cancellationToken">Cancellation Token to cancel the request.</param>
     /// <returns>Task of CreateStoreResponse</returns>
-    public async Task<CreateStoreResponse> CreateStore(CancellationToken cancellationToken = default) {
+    public async Task<CreateStoreResponse> CreateStore(CreateStoreRequest body, CancellationToken cancellationToken = default) {
         var pathParams = new Dictionary<string, string> {};
 
         var queryParams = new Dictionary<string, string>();
@@ -216,10 +217,11 @@
     /// List all stores Returns a paginated list of OpenFGA stores and a continuation token to get additional stores. The continuation token will be empty if there are no more stores. 
     /// </summary>
     /// <exception cref="OpenFga.Sdk.Exceptions.ApiException">Thrown when fails to make API call</exception>
+    /// <param name="pageSize"> (optional)</param>
     /// <param name="continuationToken"> (optional)</param>
     /// <param name="cancellationToken">Cancellation Token to cancel the request.</param>
     /// <returns>Task of ListStoresResponse</returns>
-    public async Task<ListStoresResponse> ListStores(string? continuationToken = default(string?), CancellationToken cancellationToken = default) {
+    public async Task<ListStoresResponse> ListStores(int? pageSize = default(int?), string? continuationToken = default(string?), CancellationToken cancellationToken = default) {
         var pathParams = new Dictionary<string, string> {};
 
         var queryParams = new Dictionary<string, string>();
