--- clients/fga-python-sdk/openfga_sdk/api/open_fga_api.py
+++ open_fga_api.py
@@ -501,13 +501,15 @@
             local_var_params=local_var_params,
         )

-    async def create_store(self, **kwargs):
+    async def create_store(self, body, **kwargs):
         """Create a store

         Create a unique OpenFGA store which will be used to store authorization models and relationship tuples.

-        >>> thread = await api.create_store()
-
+        >>> thread = await api.create_store(body)
+
+        :param body: (required)
+        :type body: CreateStoreRequest
         :param async_req: Whether to execute the request asynchronously.
         :type async_req: bool, optional
         :param _preload_content: if False, the urllib3.HTTPResponse object will
@@ -524,15 +526,17 @@
         :rtype: CreateStoreResponse
         """
         kwargs["_return_http_data_only"] = True
-        return await self.create_store_with_http_info(**kwargs)
-
-    async def create_store_with_http_info(self, **kwargs):
+        return await self.create_store_with_http_info(body, **kwargs)
+
+    async def create_store_with_http_info(self, body, **kwargs):
         """Create a store

         Create a unique OpenFGA store which will be used to store authorization models and relationship tuples.

-        >>> thread = api.create_store_with_http_info()
-
+        >>> thread = api.create_store_with_http_info(body)
+
+        :param body: (required)
+        :type body: CreateStoreRequest
         :param async_req: Whether to execute the request asynchronously.
         :type async_req: bool, optional
         :param _return_http_data_only: response data without head status code
@@ -561,7 +565,7 @@
         local_var_params = locals()

         all_params = [
-
+            'body'
         ]
         all_params.extend(
             [
